packages/custom-elements/.clang-format

```diff
@@ -0,0 +1,12 @@
+BasedOnStyle: Google
+AlignAfterOpenBracket: AlwaysBreak
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: false
```

Comment:
Can we enforce braces for all blocks too? I've always considered 'single line' a prereq for 'no braces'.

Reply:
That's an eslint setting. Looks like eslint is inconsistently applied across the polyfills, I'll send over a follow up PR that applies it to all the polyfills (save url, which is weird), and I've backported the eslint fixes for custom elements into this PR.